wolox-training/mr-rails
# Pundit

# Summary
* Added Pundit for user authorization

# Test plan
1. Go to /articles
2. You should not be able to edit articles not owned by you
3. Open an article
4. You should not be able to delete comments not owned by you

# Trello card
https://trello.com/c/RNVL2KZi/16-instalar-y-configurar-pundit
